plat/common/include/x86/cpu.h

```diff
@@ -31,7 +31,6 @@
 #define __PLAT_COMMON_X86_CPU_H__
 
 #include <uk/arch/types.h>
-#include <x86/cpu_defs.h>
```

Comment:
As a general rule, all files must include headers they depend on. For example, if a file uses a register definition, it must *explicitly* include the header that contains that definition, or *implicitly* include it, by including a header that includes that header.

In this case, this file uses register definitions like `CPUID3` that you moved to `arch`. You should therefore include the header that defines these, i.e. `#include <uk/arch/lcpu.h>`.